### PR TITLE
Make `AQTiCCXGate` a class

### PR DESCRIPTION
**Pull request by vtomole**

### CODE REVIEW

File: qiskit_superstaq/custom_gates.py [hunk 1]
```diff
@@ -221,6 +221,11 @@ def __str__(self) -> str:
         return f"ICCXGate(label={self.label}, ctrl_state={self.ctrl_state})"
 
 
+class IICCX(ICCXGate):
+    def __init__(self) -> None:
+        super().__init__(ctrl_state="00")
```
Comment (richrines1) [inline qiskit_superstaq/custom_gates.py:226]:
```suggestion
    def __init__(self, label: Optional[str] = None) -> None:
        super().__init__(label=label, ctrl_state="00")
```

**Review by richrines1 — COMMENTED** [review]
in custom_resolver:
```
if gate.name == "iccx_o0":
        return IICCXGate(label=gate.label)
```

**Comment by vtomole** [comment]
> in custom_resolver:
```
if gate.name == "iccx_o0":
        return IICCXGate(label=gate.label)
```

Done.

File: qiskit_superstaq/__init__.py [hunk 2]
```diff
@@ -6,10 +6,10 @@
 from ._version import __version__
 from .custom_gates import (
     AceCR,
+    AQTiCCXGate,
+    AQTiToffoliGate,
     ICCXdgGate,
     ICCXGate,
```
Comment (richrines1) [inline qiskit_superstaq/__init__.py:12]: i believe these were not included in #154
```suggestion
```

File: qiskit_superstaq/__init__.py [hunk 3]
```diff
@@ -19,11 +19,11 @@
     "AceCR",
     "API_URL",
     "API_VERSION",
+    "AQTiCCXGate",
+    "AQTiToffoliGate",
     "compiler_output",
     "ICCXGate",
     "ICCXdgGate",
```
Comment (richrines1) [inline qiskit_superstaq/__init__.py:26]:
```suggestion
```

File: qiskit_superstaq/custom_gates.py [hunk 4]
```diff
@@ -258,7 +263,7 @@ def __str__(self) -> str:
 
 
 ITOFFOLIGate = ICCXGate
```
Comment (richrines1) [inline qiskit_superstaq/custom_gates.py:265]:
```suggestion
```

**Review by richrines1 — COMMENTED** [review]
this seems to be missing most of the changes to custom_gates_test.py and serialization_test.py in #154?

**Comment by vtomole** [comment]
> this seems to be missing most of the changes to custom_gates_test.py and serialization_test.py in https://github.com/SupertechLabs/qiskit-superstaq/pull/154?

Re-added them.

File: qiskit_superstaq/custom_gates.py (hunk 4 above)
Comment (vtomole) [reply qiskit_superstaq/custom_gates.py]: Done.

File: qiskit_superstaq/__init__.py (hunk 3 above)
Comment (vtomole) [reply qiskit_superstaq/__init__.py]: Done.

File: qiskit_superstaq/__init__.py (hunk 2 above)
Comment (vtomole) [reply qiskit_superstaq/__init__.py]: Done.

File: qiskit_superstaq/custom_gates.py (hunk 1 above)
Comment (vtomole) [reply qiskit_superstaq/custom_gates.py]: Done.

**Pull request merged**